CHANGELOG

```diff
@@ -1,3 +1,8 @@
+* Thu Oct 11 2018 Nick Miller <nick.miller@onyxpoint.com> - 6.0.7-0
```

Comment:
This is an api change, which should be a minor version bump.  Please change the changelog entries for 6.0.7 to match the fixed changelog entry